test/integration/apiserver/apiserver-0.config

```diff
@@ -1,2 +1,2 @@
-/07cf43c0-e32f-47df-af11-52e3cbdf95fa/registry/pods/, tenant1, tenant2
-/07cf43c0-e32f-47df-af11-52e3cbdf95fa/registry/replicasets/, tenant1, tenant2
+/fc1161b6-dbfc-48a6-b703-acab227e164b/registry/pods/, tenant1, tenant2
```

Comment:
The changes to test/integration/apiserver/apiserver-0.config and apiserver-1.config during test should not be checked in.

Reply:
Rolled back